### DISCUSSION TITLE
Database Location

### DISCUSSION THREAD

**Discussion by drewauff** [tldr-pages/tldr#7013] [Q&A] [answered]
Is there a central "database" that can be accessed other than the "pages" directories?

I am writing a small fzf/zle script to make it faster.

For  this to work I need to be able to access the database as that will be used to display the commands tldr output  in the right hand panel.

Thankyou
-
Andy

**Comment by CleanMachine1** [comment]
Are you referring to wanting a long list of every page, because we do make one of them

Its in a JSON format

https://github.com/tldr-pages/tldr-pages.github.io/blob/main/assets/index.json

**Comment by navarroaxel** [comment]
If you check the client specification you have a reference to the ZIP file that is generated and it's the recommended solution https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#caching

**Comment by sbrl** [comment]
We also have tldr.zip: <https://github.com/tldr-pages/tldr-pages.github.io/raw/master/assets/tldr.zip>

See also our [client specification](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md).

**Comment by CleanMachine1** [comment]
Hi, do any of these responses answer your question?

**Comment by drewauff** [comment]
Not really, I had to switch to tealdeer, they have a flag to list all availble..

**Comment by marchersimon** [comment, marked as answer]
Oh. so you were trying to list all pages using your client? Then that's a client-specific feature, as you already figured out.
